x-pack/plugins/security_solution/public/management/pages/policy/view/policy_details.tsx

```diff
@@ -245,6 +251,17 @@ export const PolicyDetails = React.memo(() => {
         <MacEvents />
         <EuiSpacer size="l" />
         <LinuxEvents />
+
+        <EuiButtonEmpty data-test-subj="advancedPolicyButton" onClick={handleAdvancedPolicyClick}>
```

Comment:
I think you need a spacer above this button - when you look at it in the browser while it is focused, it shows that its top border is actually "blued" to the prior container (Linux Events).

Reply:
c7054c6fba0392e309a80f9ccde2823183ea8412